### ISSUE TITLE
gpg vs gpg2 differences can cause issue in some specific cases

### ISSUE THREAD

**Issue by balos1** [balos1/easy-gpg-to-paper#1]
There are systems that currently have two GnuPG versions at the same time, with binaries named gpg and gpg2. In some specific cases, like using x25519 keys, the script fails because of this.

**Comment by balos1** [comment]
First off, I still need to explore that actual issue more.

One possible solution is to use the official GnuPG python bindings. However, the con to this is that the bindings are a pain to build and install. Prebuilt binaries of easy-gpg-to-paper would be a necessity in this case to maintain "easy".

A better solution is to probably pipe a key directly into easy-gpg-to-paper for exporting and to pipe a reassembled key to stdout for importing. Then the script is totally independent of gpg. Usage of the script would then be something like:

`gpg --export-secret-key | gpg2paper.py export -png -o my-key.png`
`gpg2paper.py import -png -i my-key*.png | gpg --import`

**Comment by DancingQuanta** [comment]
I perfer the piping method. This should allow the user to take in account of their current gpg software. On Ubuntu you usually have `gpg` and `gpg2` together, which often confuse various programs to no end.

**Comment by balos1** [comment]
@DancingQuanta I agree. I plan on adding this feature in the coming weeks. If you have any suggestions or requests besides this, feel free to open up an issue. Thank you for your input!
